sohamb117.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "@sohamb117"
+project: "Custom Locus Keyboard"
+---
+
+# Modified Locus Keyboard
+
+## Summary
+
+I'm going to build a split 45% keyboard using a stack of cut acrylic parts. The goal is to build a completely different type of keyboard from the one I am using right now, and to practice improve my skills at soldering, 3D printing, silicone casting, and modification. I'm excited to work on a completely different (and far more in-depth) project than most of my other hardware work, as this will require not just computer science skills but also a knowledge of materials and manufacturing.
+
+## Plan
+
+1) Go to Home Depot and compare different plastics based on their flexibility and resonance (this is just to decide what material to use)
+2) Submit an order for the laser-cut case
+3) 3D print tracing plates for the foam inserts, as well as a mold for the silicone
+4) Cut the foam using the 3d-printed plates
+5) Pour the silicone into the mold, let it harden, and then remove it
+6) Modify PCB to add flex cuts, using a 3d-printed map and a Dremel (this is risky, so I might not do it, depending on how flexible the PCB is in-hand)
```

Comment:
The issue is that PCB design for a keyboard has to be _really_ precise because it needs to fit the case and the switch plate, and also adding flex cuts drastically increases the price.